spec/jobs/stream_notifications_job_spec.rb

```diff
@@ -0,0 +1,31 @@
+RSpec.describe StreamNotificationsJob do
+  describe '#perform' do
+    context 'with zero users' do
+      let(:users) { nil }
+
+      it 'does not broadcast' do
+        expect(Hyrax::NotificationsChannel).not_to receive(:broadcast_to)
+        described_class.perform_now(users)
+      end
+    end
+
+    context 'with a user' do
+      let(:users) { create(:user) }
+      let(:mailbox) { double('mailbox', unread_count: 7, label: "You've got mail!") }
+
+      before do
+        allow(UserMailbox).to receive(:new).and_return(mailbox)
+      end
+
+      it 'broadcasts to the user' do
+        # TODO: figure out why the heck this test fails
```

Comment:
Sorry to wade in, just following developments. Is this still valid? Your CI build is passing.

Reply:
Ah! This is cruft. Will remove, thx!

Reply:
Done.